test/functional/shared.js

```diff
@@ -104,6 +104,9 @@ function invalidAssets (account, option, badTransactions) {
 			case 'dapp':
 				transaction = node.lisk.dapp.createDapp(account.password, null, node.guestbookDapp);
 				break;
+			case 'inTransfer':
+				transaction = node.lisk.transfer.createInTransfer(node.guestbookDapp.id, Date.now(), node.gAccount.password);
+				break;
```

Comment:
the naming here is odd. The other cases go `node.lisk.NAME`

Here we are using the name of type 0 as the base name. Should it be called intransfer instead?

Reply:
> The naming here is odd. The other cases go node.lisk.NAME.

I believe naming relates to how the `lisk-js` API is designed.

> Here we are using the name of type 0 as the base name. Should it be called intransfer instead?

This is a correct strategy, using parent/base type name, which all cases are.